File: lib/model/animal.dart
// File created by Jordan Clark

// This class is a simple Plain-Old-Data object designed to mirror the database representation of the animal.
class Animal
{
  int animalId;
  String commonName;
  String scientificName;
  String tags;
  int classid;
  int exhibitid;
  List<String> pictureURL;

  Animal(this.animalId, this.commonName, this.scientificName, this.classid, this.exhibitid, [this.tags = "", this.pictureURL])
  {
    if (pictureURL == null)
    {
      pictureURL = List<String>();
    }
  }
}

File: ZooCompanion/ZooCompanion/Models/Animal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ZooCompanion.Models
{
    public class Animal
    {
        public int AnimalId { get; set; }
        public string CommonName { get; set; }
        public string ScientificName { get; set; }
        public byte[] QRCode { get; set; }
        public string Tags { get; set; }
        public Nullable<int> ClassId { get; set; }
        public Nullable<int> ExhibitId { get; set; }
    }
}